tests/osaka/eip7692_eof_v1/eip7069_extcall/test_calls.py

```diff
@@ -1104,3 +1130,56 @@ def test_eof_calls_msg_depth(
         post=post,
         tx=tx,
     )
+
+
+@pytest.mark.parametrize("target_account_type", TargetAccountType)
+@pytest.mark.parametrize("delegate", [True, False])
+def test_extdelegate_call_targets(
+    state_test: StateTestFiller,
+    pre: Alloc,
+    target_account_type: TargetAccountType,
+    target_address: Address,
+    delegate: bool,
+):
+    """
+    Test EOF contracts extdelegatecalling various targets, especially resolved via 7702
+    delegation.
+    """
+    env = Environment()
+
+    if delegate:
+        target_address = pre.fund_eoa(0, delegation=target_address)
+
+    caller_contract = Container.Code(
+        Op.SSTORE(slot_code_worked, value_code_worked)
+        + Op.SSTORE(slot_call_result, Op.EXTDELEGATECALL(address=target_address))
+        + Op.STOP,
+    )
+
+    calling_contract_address = pre.deploy_contract(caller_contract)
+
+    tx = Transaction(
+        sender=pre.fund_eoa(),
+        to=Address(calling_contract_address),
```

Comment:
```suggestion
        to=calling_contract_address,
```
No need to cast here.